plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -165,7 +167,7 @@ global proc mayaUsdMenu_windowMenuCallback() {
 // mayaUsdMenu_windowMenuCallback
 // setup the items in Maya's "Window->General Editors" menu
 global proc mayaUsdMenu_generalEditorsMenuCallback() {
-    if (!(`menuItem -query -exists wmUsdLayerEditorMenuitem`))
+    if (`exists mayaUsdLayerEditorWindow` && !(`menuItem -query -exists wmUsdLayerEditorMenuitem`))
```

Comment:
This makes sure to not add the "USD Layer Editor" under the Maya "Windows" menu.